[2024-11-28T13:10:01.857+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-28T13:10:01.870+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_airflow_dag.get_datetime scheduled__2024-11-28T13:09:00+00:00 [queued]>
[2024-11-28T13:10:01.876+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_airflow_dag.get_datetime scheduled__2024-11-28T13:09:00+00:00 [queued]>
[2024-11-28T13:10:01.876+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-28T13:10:01.885+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): get_datetime> on 2024-11-28 13:09:00+00:00
[2024-11-28T13:10:01.894+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11223) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-28T13:10:01.895+0000] {standard_task_runner.py:72} INFO - Started process 11232 to run task
[2024-11-28T13:10:01.896+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'scheduled__2024-11-28T13:09:00+00:00', '--job-id', '32943', '--raw', '--subdir', 'DAGS_FOLDER/first_***_dag.py', '--cfg-path', '/tmp/tmpxna7t1m_']
[2024-11-28T13:10:01.897+0000] {standard_task_runner.py:105} INFO - Job 32943: Subtask get_datetime
[2024-11-28T13:10:01.936+0000] {task_command.py:467} INFO - Running <TaskInstance: first_airflow_dag.get_datetime scheduled__2024-11-28T13:09:00+00:00 [running]> on host 5365fe4338c6
[2024-11-28T13:10:01.992+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_***_dag' AIRFLOW_CTX_TASK_ID='get_datetime' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T13:09:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-28T13:09:00+00:00'
[2024-11-28T13:10:01.993+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-28T13:10:01.993+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-28T13:10:01.994+0000] {logging_mixin.py:190} INFO - Current task name:get_datetime state:running start_date:2024-11-28 13:10:01.870979+00:00
[2024-11-28T13:10:01.994+0000] {logging_mixin.py:190} INFO - Dag name:first_***_dag and current dag run status:running
[2024-11-28T13:10:01.994+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-28T13:10:01.994+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-28T13:10:01.995+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'date']
[2024-11-28T13:10:02.005+0000] {subprocess.py:86} INFO - Output:
[2024-11-28T13:10:02.008+0000] {subprocess.py:93} INFO - Thu Nov 28 13:10:02 UTC 2024
[2024-11-28T13:10:02.008+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-28T13:10:02.025+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-28T13:10:02.026+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, run_id=scheduled__2024-11-28T13:09:00+00:00, execution_date=20241128T130900, start_date=20241128T131001, end_date=20241128T131002
[2024-11-28T13:10:02.032+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-28T13:10:02.032+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-28T13:10:02.032+0000] {logging_mixin.py:190} INFO - Dag name:first_***_dag queued_at:2024-11-28 13:10:00.902838+00:00
[2024-11-28T13:10:02.032+0000] {logging_mixin.py:190} INFO - Task hostname:5365fe4338c6 operator:BashOperator
[2024-11-28T13:10:02.069+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-28T13:10:02.075+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
